Userland/Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -2177,30 +2177,30 @@ RefPtr<Gfx::Font const> StyleComputer::compute_font_for_style_values(DOM::Elemen
 
     if (font_size.is_identifier()) {
         // https://w3c.github.io/csswg-drafts/css-fonts/#absolute-size-mapping
-        constexpr auto get_absolute_size_mapping = [](Web::CSS::ValueID identifier) {
+        auto get_absolute_size_mapping = [](Web::CSS::ValueID identifier) -> CSSPixels {
```

Comment:
Shouldn't a PixelFraction be nicer here?

Reply:
Yeah, it definitely should be a fraction, good catch!